dask/dataframe/tests/test_rolling.py

```diff
@@ -36,6 +39,27 @@ def rolling_tests(p, d):
     # Test with kwargs
     eq(pd.rolling_sum(p, 3, min_periods=3), dd.rolling_sum(d, 3, min_periods=3))
 
+    # New rolling API
+    eq(p.rolling(3).count(), d.rolling(3).count())
+    eq(p.rolling(3).sum(), d.rolling(3).sum())
+    eq(p.rolling(3).mean(), d.rolling(3).mean())
+    eq(p.rolling(3).median(), d.rolling(3).median())
+    eq(p.rolling(3).min(), d.rolling(3).min())
+    eq(p.rolling(3).max(), d.rolling(3).max())
+    eq(p.rolling(3).std(), d.rolling(3).std())
+    eq(p.rolling(3).var(), d.rolling(3).var())
+    eq(p.rolling(3).skew(), d.rolling(3).skew())
+    eq(p.rolling(3).kurt(), d.rolling(3).kurt())
+    eq(p.rolling(3).quantile(0.5), d.rolling(3).quantile(0.5))
+    eq(p.rolling(3).apply(mad), d.rolling(3).apply(mad))
+    with ignoring(ImportError):
+        eq(p.rolling(3, win_type='boxcar').sum(),
+           d.rolling(3, win_type='boxcar').sum())
```

Comment:
Why was the ImportError here ignored originally?  If it is for older pandas versions then we probably don't need to worry about this, given the dependence on recent pandas for this feature. 

Conversely, we may expect this test to fail under the older Pandas version tests (do we still want to support these?)